### PR TITLE
Add SDK

### LINKED ISSUE #20: Add Appwrite SDK

### PR DESCRIPTION
**Pull request by lenzfliker**
Fixed #20

### CODE REVIEW

**Review by king-11 — APPROVED** [review]

**Comment by king-11** [comment]
:+1:  LGTM

File: README.md [hunk 1]
```diff
@@ -93,4 +94,17 @@ Share your apps here! send a pull request!
...
+* [Appwrite Python SDK](https://github.com/appwrite/sdk-for-python)
+* [Appwrite Flutter SDK](https://github.com/appwrite/sdk-for-flutter)
+* [Appwrite Web SDK](https://github.com/appwrite/sdk-for-js)
+* [Appwrite PHP SDK](https://github.com/appwrite/sdk-for-php)
+* [Appwrite SDK for Dart](https://github.com/appwrite/sdk-for-dart)
+* [Appwrite Node.js SDK](https://github.com/appwrite/sdk-for-node)
+* [Appwrite SDK for Go](https://github.com/appwrite/sdk-for-go)
+* [Appwrite Ruby SDK](https://github.com/appwrite/sdk-for-ruby)
+* [Appwrite Deno SDK](https://github.com/appwrite/sdk-for-deno)
+* [Appwrite Flutter (Dev Channel) SDK](https://github.com/appwrite/sdk-for-flutter-dev)
```
Comment (eldadfux) [inline README.md:110]: This repo is about to be deprecated. Please remove the link.

File: README.md [hunk 2]
```diff
@@ -93,4 +94,17 @@ Share your apps here! send a pull request!
...
+
+* [Appwrite Python SDK](https://github.com/appwrite/sdk-for-python)
+* [Appwrite Flutter SDK](https://github.com/appwrite/sdk-for-flutter)
+* [Appwrite Web SDK](https://github.com/appwrite/sdk-for-js)
+* [Appwrite PHP SDK](https://github.com/appwrite/sdk-for-php)
+* [Appwrite SDK for Dart](https://github.com/appwrite/sdk-for-dart)
+* [Appwrite Node.js SDK](https://github.com/appwrite/sdk-for-node)
+* [Appwrite SDK for Go](https://github.com/appwrite/sdk-for-go)
+* [Appwrite Ruby SDK](https://github.com/appwrite/sdk-for-ruby)
+* [Appwrite Deno SDK](https://github.com/appwrite/sdk-for-deno)
```
Comment (eldadfux) [inline README.md:109]: Please add `(beta)` label to Deno, Ruby, and Python SDKs

File: README.md [hunk 3]
```diff
@@ -93,4 +94,17 @@ Share your apps here! send a pull request!
...
\ No newline at end of file
+* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
+
+## SDK
+
+* [Appwrite Python SDK](https://github.com/appwrite/sdk-for-python)
+* [Appwrite Flutter SDK](https://github.com/appwrite/sdk-for-flutter)
+* [Appwrite Web SDK](https://github.com/appwrite/sdk-for-js)
+* [Appwrite PHP SDK](https://github.com/appwrite/sdk-for-php)
+* [Appwrite SDK for Dart](https://github.com/appwrite/sdk-for-dart)
```
Comment (eldadfux) [inline README.md:105]: Please remove the link to Dart. This is still work in progress.

File: README.md [hunk 4]
```diff
@@ -93,4 +94,17 @@ Share your apps here! send a pull request!
...
 * [Facebook](https://www.facebook.com/appwrite.io/) (Official, global)
 * [Appwrite Developers on Facebook](https://www.facebook.com/groups/appwrite.developers/?source_id=227046654430085) (Official, global)
-* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
\ No newline at end of file
+* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
+
+## SDK
+
+* [Appwrite Python SDK](https://github.com/appwrite/sdk-for-python)
+* [Appwrite Flutter SDK](https://github.com/appwrite/sdk-for-flutter)
```
Comment (eldadfux) [inline README.md:102]: Please move Web and Flutter SDKs to the top of the list.

File: README.md [hunk 5]
```diff
@@ -10,6 +10,7 @@ Table of Contents:
   - [Showcase (Source Code)](#showcase-built-with-appwrite-)
   - [Code Examples](#code-examples)
   - [Communities](#communities)
+  - [SDK](#SDK)
```
Comment (eldadfux) [inline README.md:13]: Move this section above `Communities`

File: README.md [hunk 6]
```diff
@@ -93,4 +94,17 @@ Share your apps here! send a pull request!
 * [Twitter](https://twitter.com/appwrite_io) (Official, global)
 * [Facebook](https://www.facebook.com/appwrite.io/) (Official, global)
 * [Appwrite Developers on Facebook](https://www.facebook.com/groups/appwrite.developers/?source_id=227046654430085) (Official, global)
-* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
\ No newline at end of file
+* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
+
+## SDK
```
Comment (eldadfux) [inline README.md:99]: Move this section above `Communities`

**Pull request closed without merging**